src/AspNetCoreModuleV2/IISLib/ntassert.h

```diff
@@ -11,6 +11,10 @@
     #undef ASSERT
 #endif
 
+#ifdef DEBUG
```

Comment:
Can we have this in debugutil?

Reply:
Why?

Reply:
Nvrm I don't think it would work defining it elsewhere. I just wanted our debug stuff to be centralized.